README.adoc

```diff
@@ -237,13 +218,9 @@ the setup method creates a JAX-RS client, which makes HTTP requests to the `inve
 client must also be registered with a JSON-P provider to process JSON resources.
 The teardown method simply destroys this client instance as well as the HTTP responses.
 
-Let’s define the test case:
-
-* The `testFallbackForGet()` test case sends a request to the `inventory` service to get the systems properties for a
-host name when the `system` service is available. Then, it puts the `system` service on maintenance.
-Afterward, it makes a second request to the `inventory` service to return the system properties for that
-host name. This time the `inventory` service calls the fallback method to return the cached properties.
-The test case then asserts whether the system properties that returned from the first request were more than the second.
+The `testFallbackForGet()` test case sends a request to the `inventory` service to get the systems properties for a
+host name before and after the `system` service becomes unavailable. Then it asserts outputs from the two requests
+that they are different from each other.
```

Comment:
I'd change the last sentence to, "Then, it asserts outputs from the two requests to ensure that they are different from each other."

Reply:
Fixed